dags/sync_dist.py

```diff
@@ -64,44 +65,42 @@
     t3 = BashOperator(
         task_id='pull',
         bash_command='cd ~/airflow/run/auto-news/src && python3 af_sync.py '
-        '--start {{ ds }} '
-        '--prefix=./run '
-        '--run-id={{ run_id }} '
-        '--job-id={{ ti.job_id }} '
-        '--data-folder=data/sync/{{ ds }} '
-        '--sources={{ dag_run.conf.setdefault("sources", "Twitter,Article,Youtube,RSS,Reddit") }} '
+                     '--start {{ ds }} '
+                     '--prefix=./run '
+                     '--run-id={{ run_id }} '
+                     '--job-id={{ ti.job_id }} '
+                     '--data-folder=data/sync/{{ ds }} '
     )
 
     t4 = BashOperator(
         task_id='save',
         bash_command='cd ~/airflow/run/auto-news/src && python3 af_dist.py '
-        '--start {{ ds }} '
-        '--prefix=./run '
-        '--run-id={{ run_id }} '
-        '--job-id={{ ti.job_id }} '
-        '--data-folder=data/sync '
-        '--sources={{ dag_run.conf.setdefault("sources", "Twitter,Article,Youtube,RSS,Reddit") }} '
-        '--targets={{ dag_run.conf.setdefault("targets", "Milvus") }} '
-        '--min-rating={{ dag_run.conf.setdefault("min-rating", 4) }} '
-        '--dedup={{ dag_run.conf.setdefault("dedup", True) }} '
+                     '--start {{ ds }} '
+                     '--prefix=./run '
+                     '--run-id={{ run_id }} '
+                     '--job-id={{ ti.job_id }} '
+                     '--data-folder=data/sync '
+                     '--targets={{ dag_run.conf.setdefault("targets", "Milvus") }} '
+                     '--min-rating={{ dag_run.conf.setdefault("min-rating", 4) }} '
+                     '--dedup={{ dag_run.conf.setdefault("dedup", True) }} '
```

Comment:
PR contains a lot of formatting changes, and it looks like those are due to different tab-indent lengths, I feel that you are using a different indent, "tab == 8 spaces", or even longer, we may consider minimizing this kind of changes as much as possible.